examples/update/dfx.json

```diff
@@ -1,14 +1,28 @@
 {
     "canisters": {
-        "update": {
+        "azle": {
             "type": "custom",
-            "build": "npx azle update",
-            "root": "src",
-            "ts": "src/update.ts",
-            "candid": "src/update.did",
-            "wasm": "target/wasm32-unknown-unknown/release/update.wasm",
+            "build": "npx azle azle",
+            "root": "canisters/azle",
+            "ts": "canisters/azle/index.ts",
+            "candid": "canisters/azle/index.did",
+            "wasm": "target/wasm32-unknown-unknown/release/azle.wasm",
             "declarations": {
-                "output": "test/dfx_generated/update"
+                "output": "dfx_generated/azle"
+            }
+        },
+        "motoko": {
+            "main": "canisters/motoko/main.mo",
```

Comment:
Let's keep the dfx.json files using the same conventions, the motoko should have type: motoko, and the rust should have type: rust first, then package, then candid